.changeset/nice-masks-invite.md

```diff
@@ -0,0 +1,24 @@
+---
+"@comet/admin-theme": major
+"@comet/cms-admin": major
+"@comet/admin": major
+---
+
+Bump @mui/x-data-grid peer dependency to v7
+
+This has breaking changes in DataGrid.
+Follow the official [migration guide](<(https://mui.com/x/migration/migration-data-grid-v6/)>) to upgrade.
+
+As well, be aware if you have a date in the data grid, you will need to add a `valueGetter`
+
+```diff
+    <DataGrid
+        //other props
+        columns=[
+        {
+            field: "updatedAt",
+            type: "dateTime",
++            valueGetter: (params, row) => row.updatedAt && new Date(row.updatedAt)
```

Comment:
FYI: A helper for the `date` and `dateTime` columns will be added to `v7` which includes this by default: https://github.com/vivid-planet/comet/pull/2628/files#diff-efa1a5fb8c374417853237b343a64cf3939a6f7289099a584de133856a05dac4R4